docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,69 @@
+
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
+
+## Discovering deprecated APIs
+
+### *What is deprecated API*
+*The .NET Framework is a large product that is getting constantly upgraded to better serve customer needs. It is natural to deprecate some APIs and replace them with new ones. Previously old APIs were marked with `[Obsolete]` attribute. There was only one diagnostic ID (CS0612) for all obsolete APIs that lead to one description for all cases and ability to suppress warnings for either all cases or none of them. To provide information and a way to suppress a warning per each API or a class of APIs, `Deprecation` term was introduced.*
+
+When a deprecated API, such as \<xref:System.Net.WebClient>, is used in a code, API Analyzer highlights it with a green squiggly line and shows a light bulb on the left as in the following example:
+
+!["Screenshot of WebClient API with green squiggly line and light bulb on the left"](media/api-analyzer/green-squiggle.jpg)
+
+The Error List window contains warnings with a unique ID per deprecated API (in the following example `DE004`). By clicking on the ID, you go to a webpage with detailed information about why the API was deprecated and suggestions regarding alternative APIs that can be used.
+
+!["Screenshot of the Error List window showing warning's ID and description"](media/api-analyzer/warnings.jpg)
+
+Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Quick Actions and Refactorings**. Here you get two suppression options: locally (in source) or globally (in a suppression file). We encourage developers to use global suppression to ensure consistency of API usage across their project.
+
+## Discovering cross-platform issues
+Similar to deprecated APIs the analyzer squiggles out all APIs that are not cross-platform supported (for example `Console.WindowWidth` works on Windows but not on Linux and macOS). The diagnostic ID is shown in Error List window. You can suppress warning by right click and choosing "Quick Actions and Refactorings". Unlike deprecation cases where you have two options: either keep using deprecated member and suppress warnings or not use it at all, here if you are developing your code only for certain platform you can suppress all warnings for all other  platforms you don't plan to run your code on. To do so, you just need to edit your project file and add a property `PlatformCompatIgnore` that lists all platforms to be ignored:
```

Comment:
missing comma after "Similar to deprecated APIs" and "for example"

missing article on "You can supress warning.." Should it be You can suppress that warning?

Bold the Quick Actions and Refactorings since it's a UI option instead of quotes

I'd put the info on the two options in parentheses, the : breaks the flow a bit

remove extra space in for all other  platforms

a property `PlatformCompatIgnore` -> a `PlatformCompatIgnore` property